board/boards/dos.h

```diff
@@ -49,23 +49,8 @@ void dos_set_led(uint8_t color, bool enabled) {
   }
 }
 
-void dos_set_bootkick(bool enabled){
-  set_gpio_output(GPIOC, 4, !enabled);
-}
-
-bool dos_board_tick(bool ignition, bool usb_enum, bool heartbeat_seen, bool harness_inserted) {
-  bool ret = false;
-  if ((ignition && !usb_enum) || harness_inserted) {
-    // enable bootkick if ignition seen or if plugged into a harness
-    ret = true;
-    dos_set_bootkick(true);
-  } else if (heartbeat_seen) {
-    // disable once openpilot is up
-    dos_set_bootkick(false);
-  } else {
-
-  }
-  return ret;
+void dos_set_bootkick(BootState state) {
+  set_gpio_output(GPIOC, 4, state != BOOT_BOOTKICK);
```

Comment:
note this is a change in the logic for the dos to match the tres ogic. dos goes from `(ignition && !usb_enum)` -> `(ignition && !ignition_prev)`